README.asciidoc

```diff
@@ -1,2 +1,7 @@
-TODO
-Instructions how to set up and run the code.
+== Instructions
+
+1. Get the required libraries.
```

Comment:
What are the required libraries, a short description would be good. The code has currently a specific python requirement due to the realsense library so this is also a good thing to mention.

Reply:
added list of depencencies and short description